modules/statics/src/coins.ts

```diff
@@ -338,6 +350,11 @@ const BERA_FEATURES = [
   CoinFeature.EVM_WALLET,
   CoinFeature.USES_NON_PACKED_ENCODING_FOR_TXDATA,
 ];
+const MOCA_FEATURES = [...ETH_FEATURES, CoinFeature.CUSTODY_BITGO_FRANKFURT];
+const CWBTC_FEATURES = [...ETH_FEATURES, CoinFeature.CUSTODY_BITGO_FRANKFURT];
+const TKX_FEATURES = [...ETH_FEATURES, CoinFeature.CUSTODY_BITGO_FRANKFURT];
+const MANA_FEATURES = [...ETH_FEATURES, CoinFeature.CUSTODY_BITGO_FRANKFURT];
+const ARBETH_ARB_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.CUSTODY_BITGO_FRANKFURT];
```

Comment:
Do something like [this](https://github.com/BitGo/BitGoJS/blob/e06be4e59f36a9ae56da097e3a92197e328a6842/modules/statics/src/coins.ts#L312)
Just confirm if it will be DefaultFeatures or EthFeatures for the eth tokens.

Reply:
This way its extendable for ungating for other Trusts as well. For example, MANA needs to be ungated after sometime for CUSTODY_BITGO_SINGAPORE as well.

Reply:
Done, have confirmed its EthFeatures for eth tokens, as EthFeatures is inheriting from DefaultFeatures.